pkgs/development/libraries/imgui/default.nix

```
@@ -70,7 +70,7 @@ stdenv.mkDerivation rec {
     owner = "ocornut";
     repo = "imgui";
     rev = "v${version}";
```

```suggestion
    rev = "refs/tags/v${version}";
```
If we're in here doing a manual update.